<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Borrow</class>
 <widget class="QWidget" name="Borrow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="user_id">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>210</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="book_id">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>270</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="borrow_btn">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>330</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Berlin Sans FB Demi</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Borrow</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Berlin Sans FB Demi</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>User ID:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>270</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Berlin Sans FB Demi</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Book ID:</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="MessageBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>441</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Berlin Sans FB Demi</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="plainText">
    <string>                               input user id and book id:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="renew_btn">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>330</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Berlin Sans FB Demi</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Renew</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>borrow_btn</sender>
   <signal>clicked()</signal>
   <receiver>Borrow</receiver>
   <slot>borrow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>renew_btn</sender>
   <signal>clicked()</signal>
   <receiver>Borrow</receiver>
   <slot>renew()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>345</y>
    </hint>
    <hint type="destinationlabel">
     <x>469</x>
     <y>303</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>borrow()</slot>
  <slot>renew()</slot>
 </slots>
</ui>
